content/articles/how-to-make-a-discord-bot-using-python/index.md

```diff
@@ -0,0 +1,292 @@
+
+To develop a new application, Proceed towards the application's page.
+
+Select **New Application** Button.
+
+![NewApp](./img2.png)
+
+Assign a name to the application and then select "Create."
+
+![createApplication](./img3.png)
+
+Congratulations! You created a Discord account. You may view details about the application on the screen:
+
+![loggedIn](./img4.png)
+
+Remember that every software that communicates with Discord APIs, not only bots, requires a Discord application. Thus, Bot-related APIs make up a small portion of Discord's overall interface.
+
+Navigate to the "Bot" tab on the left-hand side panel, as this section is about how to build a Discord bot.
+
+3.	**Forming a Bot**
+
+A bot user listens to and automatically replies to specific events and orders on Discord, as discussed in the preceding parts.
+
+To see your code in action on Discord, you'll require to build a bot user. For that, select the "Add Bot" button.
+
+![bot](/engineering-education/how-to-make-a-discord-bot-using-python/img5.png)
+
+You'll notice the new bot user in the portal after you confirm by clicking "Yes, do it!".
+
+![bot2](./img6.png)
+
+It's important to realize that the bot user will preserve the name of your application by default. 
+
+Now that the bot is finally ready to go, where will it go?
+
+ If a bot user isn't communicating with other users, it's useless. So you'll then create a guild for your bot so that it may communicate with other people.
+
+4.**Forming a Guild (a Server)**
+
+A guild (as it's referred to in Discord's user interface) is a collection of routes where members may communicate.
+
+**Notes: While the terms guild and server are equivalent, we will use the term "guild" in this article because the APIs use the exact phrase. We will only use the term "server" in the graphical user interface to refer to a guild.**
+
+Now, go to your Discord home (link) page and establish a guild:
+
+You may see and connect with friends, direct messages, and guilds through this home page. To Add a Server, click the "+" button from the left side panel of the web page:
+
+![AddFriend](./img7.png)
+
+On the upcoming screen, select "Create a server" and give the server a name. Then, you can interact with yourself or request some people to communicate with you after the server is up and running. We'll be inviting our bot to communicate with us soon. 
+
+You'll be able to view the users on the right side and the channels on the left panel when you've done building your guild:
+
+![screen](./img8.png)
+
+Further, the final stage is to register the bot with the new guild at Discord.
+
+### Create a discord bot using discord.py library
+
+The bot's programming will be written in Python using the discord.py package. discord.py is a Discord API wrapper that makes creating a Discord bot in Python much more effortless.
+
+
+#### **Installing** `discord.py`  and creating a Repl
```

Comment:
Remove bold in titles. Use H3 and beyond to suitably identify the sections and sub-sections.